src/util/get-files.ts

```diff
@@ -168,7 +169,7 @@ export async function staticFiles(
   } else {
     // The package.json `files` whitelist still
     // honors ignores: https://docs.npmjs.com/files/package.json#files
-    const search_ = ['.'];
+    const search_ = src ? [src] : ['.'];
```

Comment:
@TooTallNate @leo I wonder why this was hardcoded to an array with a single `.`

Could we avoid looping since there is only ever one element?

Reply:
Looks like an artifact of #1126

Reply:
After reviewing the git history, I think it is safe to change this to:

```ts
const pattern = src ? src : '.';
const search = await glob(pattern, { cwd: path, absolute: true, dot: true });
```

Then we don't need to loop or copy the array.